src/EulerHS/KVConnector/Utils.hs

```diff
@@ -159,11 +162,11 @@ getPKeyAndValueList table = do
       keyValueList = sortBy (compare `on` fst) k
       rowObject = A.toJSON table
   case rowObject of
-    A.Object hm -> foldl' (\ acc x -> (go hm x) : acc) [] keyValueList
+    A.Object hm -> DL.foldl' (\acc x -> [go hm x] <> acc) [] keyValueList
```

Comment:
why not `cons`?

Reply:
i'm sure i probably had a better reason at the time, but right now looking at this i can only think of better semantics. 

`<>` in general means combine these, and it's just clearer at a glance (to my eyes).
`cons` also should do fine and probably be a bit more performant in this particular case if the function is run a lot.

Reply:
This function does get invoked for every DB call IIRC. so can we move this to use `cons`